documentation/source/simulator_support.rst

```diff
@@ -81,6 +81,15 @@ If your design's clocks vary in precision, the performance of the simulation can
 
     COCOTB_HDL_TIMEPRECISION = 1us # Set precision to 10^-6s
 
+Coverage
+--------
+
+To enable HDL code coverage, add Verilator's coverage option(s) to the :make:var:`EXTRA_ARGS` make variable, for example:
+
+ .. code-block:: make
+
+     EXTRA_ARGS += --coverage
+
 .. versionadded:: 1.3
```

Comment:
```suggestion
.. versionadded:: 1.3

Coverage
--------

To enable HDL code coverage, add Verilator's coverage option(s) to the :make:var:`EXTRA_ARGS` make variable, for example:

 .. code-block:: make

    EXTRA_ARGS += --coverage
     
This will result in coverage data being written to ``coverage.dat``.
```